### ISSUE TITLE
How to generate using your own sketches?

### ISSUE THREAD

**Issue by adelorenz** [PeterWang512/GANSketching#11]
Is it possible to generate cats by providing my own sketches? Looking at the code it seems that all sketches are hardcoded, but maybe I'm missing something.

**Comment by PeterWang512** [comment]
Yes, feel free to create you own sketches using some drawing softwares and save them into png files. You can select a similar stroke size as the ones in the released dataset

**Comment by adelorenz** [comment]
Glad to hear that, but how? Taking a look at generator.py I'm not able to understand how I can pass my own sketches. I would like to draw my own cat and apply your model, but how to do it is not described (or, probably, I'm missing something)

**Comment by PeterWang512** [comment]
Actually, you might want to look into our example training script. For example, this one: https://github.com/PeterWang512/GANSketching/blob/main/scripts/train_authorsketch_ffhq0.sh

Changing `--dataroot_sketch` to your own sketches could work. You need to make a folder of your own sketches, and to do that you can check how the released dataset is organized.

**Comment by adelorenz** [comment]
Tnx, I will check and let you know!

Il gio 17 feb 2022, 00:46 Sheng-Yu Wang ***@***.***> ha
scritto:

**Comment by gorkeygorkem** [comment]
Did you manage to do it? If you did, can you explain how?

**Comment by PeterWang512** [comment]
For example, for the models shown in the teaser figure, we created the sketches using MS paint. But you can use other sketching tool to create your own sketches.
